ui/js/controllers/perf/graphs.js

```diff
@@ -628,11 +628,13 @@ perf.controller('GraphsCtrl', [
         function addSeriesList(partialSeriesList) {
             $scope.loadingGraphs = true;
             return $q.all(partialSeriesList.map(function (partialSeries) {
+                var param = { framework: partialSeries.frameworkId};
```

Comment:
`var param` -> `let params` (note the plural)

Also pay attention to the spacing in the curly brackets: should be one after `frameworkId`. There is probably an eslint rule we should enable for this.